oceanmesh/mesh_generator.py

```diff
@@ -74,25 +147,25 @@ def generate_mesh(domain, edge_length, **kwargs):
         See below
 
     :Keyword Arguments:
-        * *bbox* (``tuple``) --
-            Bounding box containing domain extents. REQUIRED IF NOT USING :class:`edge_length`
+        * *bbox* (``tuple`` or ``list of tuples``) --
+            Bounding box(es) containing domain extents. REQUIRED IF NOT USING :class:`edge_length`
         * *nscreen* (``float``) --
             Output to the screen `nscreen` timestep. (default==1)
         * *max_iter* (``float``) --
             Maximum number of meshing iterations. (default==50)
         * *seed* (``float`` or ``int``) --
             Psuedo-random seed to initialize meshing points. (default==0)
-        * *perform_checks* (`boolean`) --
-            Whether or not to perform mesh linting/mesh cleanup. (default==False)
         * *pfix* (`array-like`) --
             An array of points to constrain in the mesh. (default==None)
         * *axis* (`int`) --
             The axis to decompose the mesh (1,2, or 3). (default==1)
         * *verbose* (``int``) --
             Output to the screen `verbose` (default==1). If `verbose`==1 only start and end messages are
+            If `verbose`==2 all messages are displayed.
         * *min_edge_length* (``float``) --
-            The minimum element size in the domain. Taken from `domain` normally.
-
+            The minimum element size in the domain.
+        * *plot* (``int``) --
```

Comment:
plot pause should be infinite and then in `generate_mesh` the appearance should last for a short duration.